### PR TITLE
[WIP] Update/docs

### PR DESCRIPTION
**Pull request by Kennedy242**
- Update image for months selector
- Remove extra semicolon

### CODE REVIEW

File: static/img/tips-tricks/months-selector.png
Comment (youngcw) [inline static/img/tips-tricks/months-selector.png:1]: I think all the other images in the docs use the light theme.  Can you add a light version of this?

Comment (Kennedy242) [reply static/img/tips-tricks/months-selector.png:1]: Yep! Done

**Review by youngcw — APPROVED** [review]
Thanks.  And as an FYI, remove the [WIP] label if you are ready for a review.

**Comment by Kennedy242** [comment]
> Thanks. And as an FYI, remove the [WIP] label if you are ready for a review.

Thank you for telling me

**Pull request merged**